### PR TITLE
chore(cdk/a11y): fix contrast spelling

### PR DESCRIPTION
**Pull request by SchnWalter**

### CODE REVIEW

**Comment by SchnWalter** [comment]
@crisbeto, I just found another word with wrong spelling in several places. Should I update this MR?

**Comment by crisbeto** [comment]
Sure, go for it.

File: src/cdk/a11y/aria-describer/aria-reference.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-/** IDs are deliminated by an empty space, as per the spec. */
+/** IDs are delimited by an empty space, as per the spec. */
 const ID_DELIMINATOR = ' ';
```
Comment (SchnWalter) [inline src/cdk/a11y/aria-describer/aria-reference.ts:10]: Also, maybe this private constant, `ID_DELIMINATOR`, should be something else. Maybe `ID_SEPARATOR`?

Comment (jelbourn) [reply src/cdk/a11y/aria-describer/aria-reference.ts:10]: It should be `ID_DELIMITER` if you want to fix that in this PR

Comment (SchnWalter) [reply src/cdk/a11y/aria-describer/aria-reference.ts:10]: Done.

**Pull request merged**